datastore-api/app/core/startup.py

```diff
@@ -0,0 +1,72 @@
+from .bing import BingSearch
+from ..routers.dependencies import get_storage_connector, get_mongo_client
+from .es.connector import ElasticsearchConnector
+from .mongo import MongoClient
+from ..models.datastore import DatastoreRequest, DatastoreField
+from .config import settings
+import requests
+import time
+from fastapi import Request
+import jwt
+
+import logging
+
+
+def wait_for_up(url: str, ntries=100) -> None:
+    for _ in range(ntries):
+        try:
+            # import ipdb; ipdb.set_trace()
+            response = requests.get(url)
+            if response.status_code == 200:
+                logger.info(f"Connected to {url}")
+                break
+        except:
```

Comment:
dont put logic into exception blocks. Also, in my understanding, `requests.get` wont even throw an error if the request was unsuccessful.
```python
if response.status_code == 200:
  logger.info(f"Connected to {url}")
  return
else:
  time.sleep(1)
raise RuntimeError(f"Could not connect to {url}")
```